dist/Dockerfile.e2e/Dockerfile

```diff
@@ -9,6 +16,9 @@ WORKDIR "${HOME}/cincinnati"
 RUN mkdir -p ${HOME}/bin && \
     curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz 2>/dev/null | tar xzf - -C "${HOME}/bin/" oc
 ENV PATH="${PATH}:${HOME}/bin"
-COPY . .
+COPY --from=builder /go/src/github.com/openshift/cincinnati/target/debug/mod-* /usr/bin/
```

Comment:
Can you use `/usr/bin/cincinnati-e2e-test` here and skip the later `mv`?

Reply:
Fixed